stripe/functions/index.js

```diff
@@ -63,6 +63,22 @@ exports.createStripeCustomer = functions.auth.user().onCreate(event => {
   });
 });
 
+// Add a payment source (card) for a user by writing a stripe payment source token to Realtime database
+exports.addPaymentSource = functions.database.ref('/stripe_customers/{userId}/cards/{token}').onWrite(event => {
```

Comment:
what if we listened instead to `/stripe_customers/{userId}/cards/{pushid}/token`? or are you writing token:token to the RTDB. Guess I'm a bit confused on the schema.